stackslib/src/chainstate/nakamoto/signer_set.rs

```diff
@@ -274,7 +274,7 @@ impl NakamotoSigners {
                                 "signer".into(),
                                 Value::Principal(PrincipalData::from(signing_address)),
                             ),
-                            ("weight".into(), Value::UInt(signer.slots.into())),
+                            ("weight".into(), Value::UInt(signer.stacked_amt.into())),
```

Comment:
I think stacked_amt needs to be normalized no? Or is this already? 

I.e. shoudl be some integer from 0 to 4000?

Reply:
I was not aware that this is supposed to be normalized. I thought it was just supposed to be the amount of STX?

Reply:
This "weight" is retrieved from the reward set entry which is a number of slots value and there are only 4000 of them. If signer A has 10 slots, they get key ids 1 through 10. The next signer in the list, Signer B, has 12 slots, they get key ids 11 through 22., etc. the max key id we can have is 4000. I do not believe simply putting the stacked amt is correct.

Reply:
Ok, so I'm just going to revert this change and use the `slots` field, which seems to be the number of reward slots.

Reply:
but I'm also going to rename it to `reward-slots` to avoid confusion with stacker DB slots, which is a term also used in this file.

Reply:
Sure though weight is also fine to keep (its what is used inside reward slot entry inside the rust codebase. Would prob try to keep it consistent)

Reply:
Ok, then I'll rename the field in the struct to be `weight` then.

Reply:
Done.

Reply:
It seems there were some misunderstandings and back and forth there, but what we landed on is that this should be the number of reward slots.